src/test/java/org/openelisglobal/person/PersonServiceTest.java

```diff
@@ -1,5 +1,14 @@
 package org.openelisglobal.person;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
```

Comment:
can we stick to one style for consistency? you are using both Asset class and static assertion methods

Reply:
Thanks for your review, I will edit it now.